CHANGELOG.md

```diff
@@ -11,6 +11,15 @@ Releases are extensively tested, including the following steps.
 
 # Current releases
 
+## Prerelease 0.4.0
+
+< current main branch >
```

Comment:
I believe you will update this before merging.

Reply:
Yes, this is just a place holder until release time.